### DISCUSSION TITLE
only one route is mounted

### DISCUSSION THREAD

**Discussion by sevku** [rwf2/Rocket#1866] [Questions]
I am trying to start out with Rocket and currently following the introduction on https://rocket.rs/v0.5-rc/guide/overview/

I says that the mount method can be chained, but if I do that, it only mounts one of the routes in the chain:

``` rust
#[macro_use] extern crate rocket;

#[get("/")]
fn index() -> &'static str {
    "Hi there!"
}

#[get("/world")]
fn world() -> &'static str {
    "hello, world!"
}

#[launch]
fn rocket() -> _ {
    rocket::build()
        .mount("/hi", routes![world])
        .mount("/", routes![index])
}
```

**Comment by jebrosen** [comment]
This example works as expected for me. Both routes are listed as being mounted, and both `/` and `/hi/world` show the expected contents.

```
🛰  Routes:
   >> (index) GET /
   >> (world) GET /hi/world
```

> but if I do that, it only mounts one of the routes in the chain:

How exactly did you determine this - does the log output include both routes or not, and/or did you try accessing both routes via a browser or otherwise?

**Reply by sevku** [reply]
Thanks for your reply, it made me realize my mistake.
